react/features/app-settings/actions.js

```diff
@@ -3,7 +3,7 @@
 import { HIDE_APP_SETTINGS, SHOW_APP_SETTINGS } from './actionTypes';
 
 /**
- * Redux-signals the request to hide the app settings modal.
+ * Redux-signals the request to hide the app settings screen.
```

Comment:
One way to avoid changing this in the future would be to not use the word screen or modal and just call it app settings.

Reply:
Yeah, I'm starting to learn it :/